data/templates/load-balancing/override_haproxy.conf.j2

```diff
@@ -0,0 +1,13 @@
+[Unit]
```

Comment:
Maybe direclty add support to bind haproxy to a VRF?

Reply:
Done!
```
vyos@r14# run show conf com | match vrf
set load-balancing reverse-proxy vrf 'foo'
set vrf name foo table '1010'
[edit]
vyos@r14# 
[edit]
vyos@r14# sudo ip vrf pids foo
17806  haproxy
17808  haproxy
[edit]
vyos@r14# 

```